src/invariant/test/InvariantTestUtils.cpp

```diff
@@ -140,5 +140,40 @@ makeUpdateList(std::nullptr_t current, std::vector<LedgerEntry> const& previous)
                    });
     return updates;
 }
+
+void
+normalizeSigners(AccountEntry& acc)
+{
+    // Get indexes after sorting by acc.signer keys
+    std::vector<std::size_t> indices(acc.signers.size());
+    std::iota(indices.begin(), indices.end(), 0);
```

Comment:
`std::iota` requires to include `<numeric>` as per standard so this doesn't build for me